pipeline/packer/jenkins-worker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM jenkins/jnlp-slave:3.29-1
+
+# Need to be root to install packages
+USER root
+
+# Install Packer
+RUN wget -O packer.zip https://releases.hashicorp.com/packer/1.4.2/packer_1.4.2_linux_amd64.zip && \
+        unzip packer.zip -d /usr/bin/ && \
+        rm packer.zip
+
+# Install Terraform
+RUN wget -O terraform.zip https://releases.hashicorp.com/terraform/0.12.3/terraform_0.12.3_linux_amd64.zip && \
+        unzip terraform.zip -d /usr/bin/ && \
+        rm terraform.zip
+
+# Install Python 3.7
+RUN apt-get update && apt-get install -y build-essential checkinstall libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev libgdbm-dev libc6-dev libbz2-dev uuid-dev zlib1g-dev libffi-dev && \
+        wget -O python.tgz https://www.python.org/ftp/python/3.7.3/Python-3.7.3.tgz && \
+        tar -xf python.tgz && rm python.tgz && \
+        cd Python-3.7.3 && \
+        ./configure && make && make install && \
+        cd .. && rm -rf Python-3.7.3
```

Comment:
I did try to install a binary package, but encountered a bunch of issues doing so whilst getting the version of Python we're using (Python 3.7).

Note that I don't think I've got the _lzma module to get built. TODO-see what packages I need to install to include this as well.